scripts/deploy/bump-version.js

```diff
@@ -32,9 +32,14 @@ async function checkoutLatestMaster() {
   await exec('git pull origin master');
 }
 
+/**
+ * @param {'pre' | 'graduate'} versionType
```

Comment:
The whole file seems typed properly except this parameter which is typed as `any`, so I figured it wouldn't be too bad to add it.